lxd/db/profiles.go

```diff
@@ -170,53 +169,3 @@ DELETE FROM profiles_devices_config WHERE profile_device_id NOT IN (SELECT id FR
 
 	return nil
 }
-
-// ExpandInstanceConfig expands the given instance config with the config
-// values of the given profiles.
-func ExpandInstanceConfig(config map[string]string, profiles []api.Profile) map[string]string {
```

Comment:
Please split this into 2 commits:
1. The move of the functions and their associated usage.
2. The addition of the new argument and its associated usage.